### ISSUE TITLE
Allow only an input rule

### ISSUE THREAD

**Issue by paulburton0** [gmlaxfanatic/ItemExchange#103]
This is a suggestion or enhancement request, not an issue or bug. Since ItemExchange chests emit a redstone signal on a successful exchange, it would be nice to be able to have only an input rule without an output rule. That way, IE chests could be used to take material payments for certain redstone actions. For example, an automated toll gate that will only open when the correct material is exchanged.

**Comment by ttk2** [comment]
so right now you can't have the chest emit redstone when payment is
received?

**Comment by paulburton0** [comment]
Yes, the chest does emit a signal. Right now there has to be an input rule
and an output rule. What I was suggesting is making it so that the Redstone
signal _is_ the output rule. In other words, instead of getting a physical
entity as the product of he exchange, the result of the Redstone circuit is
the product.

Right now if I wanted to, say, charge admission to an area, I might have an
iron door with an item exchange chest next to it. If I want to charge one
iron ingot to open the door, I still have to have a "dummy" output rule in
order to make the itemexchange chest work. So I might set up the chest to
exchange 1 iron for 1 seeds and open the door with the Redstone signal on
successful transaction. It would be nice to have a "null" output rule so
that I don't have to use the dummy output rule.

---

Paul Burton
paulburton0@gmail.com
http://paul-burton.com/keys

**Comment by ttk2** [comment]
ok then, I will try to arrange that.

**Comment by erocs** [comment]
Added f415088bb8
